rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_motor_model.h

```diff
@@ -46,13 +47,16 @@ const static int CCW = 1;
 const static int CW = -1;
 } // namespace turning_direction
 
-namespace gazebo {
+enum motor_type {
```

Comment:
use enum class MotorType.

(see for instance 
https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum
or 
https://stackoverflow.com/questions/12581064/enum-vs-strongly-typed-enum)